ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -450,11 +451,14 @@ static KsqlRestApplication buildApplication(
 
     final MutableFunctionRegistry functionRegistry = new InternalFunctionRegistry();
 
+    final HybridQueryIdGenerator hybridQueryIdGenerator =
```

Comment:
nit: 

```suggestion
    final QueryIdGenerator queryIdGenerator =
```

Is less to read ;)

Reply:
I think I'd have to keep it as HybridQueryIdGenerator because StatementExecutor expects the hybrid implementation in order for it to be able to access the specific methods of the hybrid.